mobile/src/main/res/drawable/ic_image_loading_black.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:viewportWidth="24"
+    android:viewportHeight="24"
+    android:width="24dp"
```

Comment:
This picture could be way larger.
For charts is seems to be at a wrong position. I wasn't able to make a screenshot while loading the chart, so I made a red mark where it appears:
![screenshot_2018-04-24-20-03-44](https://user-images.githubusercontent.com/22525368/39205495-13b3e346-47fb-11e8-95e9-422e5ce0f3f8.png)